test/CSharpPrecompiledTestProjects/Dependency55/Dependency55.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
Can you explain this change? It shouldn't need to update - is there some issue you are working around here?

Reply:
When I switched the integration test project to multi-target, I originally got the following error message:

I didn't quite understand why it was just the Dependency 55 and 56 projects that are impacted by this?

I just changed this back to singular (`TargetFramework`) and I'm not seeing this error message anymore. 

However, I do get different results when building the sln (everything works fine):

`dotnet build WebJobs.Script.sln`

vs building the integration test project either like this (build breaks):

`dotnet build test/WebJobs.Script.Tests.Integration`

The error is always about those test projects i.e.

>/usr/local/share/dotnet/sdk/8.0.204/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(96,5): error NETSDK1013: The TargetFramework value 'net6.0'' was not recognized. It may be misspelled. If not, then the TargetFrameworkIdentifier and/or TargetFrameworkVersion properties must be specified explicitly. [/Users/likasem/source/functions/azure-functions-host/test/CSharpPrecompiledTestProjects/Dependency55/Dependency55.csproj]

>/usr/local/share/dotnet/sdk/8.0.204/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(96,5): error NETSDK1013: The TargetFramework value 'net8.0'' was not recognized. It may be misspelled. If not, then the TargetFrameworkIdentifier and/or TargetFrameworkVersion properties must be specified explicitly. [/Users/likasem/source/functions/azure-functions-host/test/CSharpPrecompiledTestProjects/WebJobsStartupTests/WebJobsStartupTests.csproj]

Thoughts?

(Also if you have any resources/docs/courses you would recommend to learn more about MSBuild and these project files, I would be very interested in learning more)

Reply:
Actually failing in the pipeline now with the last 2 commits I made :(

https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=165646&view=logs&j=18d94a44-837b-59b4-84e6-015da6d5bdc8&t=2ae8e128-7a8d-54f3-a559-ea229ccd22c0

Reply:
AHHHHHH; it's an extra `'` in `Properties="Configuration=Debug;TargetFramework=$(TargetFramework)'"` - fixing ☠️